aten/src/ATen/native/cuda/layer_norm_kernel.cu

```diff
@@ -416,7 +416,7 @@ void LayerNormBackwardKernelImpl(
   AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16,
       X.scalar_type(), "LayerNormBackwardKernelImpl", [&]() {
         LayerNormBackwardKernelImplInternal<scalar_t>(
-            dY, X, mean, rstd, gamma, M, N, dX, dgamma, dbeta);
+            dY.is_contiguous() ? dY : dY.contiguous(), X, mean, rstd, gamma, M, N, dX, dgamma, dbeta);
```

Comment:
That check is already done in the `contiguous()` call, why do you need to redo-it here?

Reply:
It used to be faster (`is_contiguous` never went through dispatch). These days `.contiguous()` avoids dispatch too in the common case, so this is no longer needed

Reply:
Will update.